api/open_saves.proto

```diff
@@ -383,6 +390,23 @@ message QueryFilter {
   Property value = 3;
 }
 
+// SortOrder is a way to order records returned by QueryRecords.
+message SortOrder {
+  // Direction is the ways to sort a list of records returned by querying.
+  enum Direction {
+    // Sort by ascending order.
+    ASC = 0;
+    // Sort by descending order.
+    DESC = 1;
+  }
+
+  // Property name to order by.
+  string property_name = 1;
+
+  // Direction to sort by (either ascending or descending).
+  Direction order = 2;
```

Comment:
I'd name it direction so that it'd look like
`something.orders[0].direction == ASC`
rather than
`something.orders[0].order == ASC`

what do you think?